lib/cmd_tool_lower.go

```diff
@@ -56,24 +57,30 @@ func CmdToolLower(
 	return nil
 }
 
-func ActionForIP(input string) {
-	fmt.Println(input)
-}
-
-func ActionForRange(input string) {
+func ActionForRange(input string) error {
 	ipRange, err := IPRangeStrFromStr(input)
 	if err != nil {
-		fmt.Println(err)
-		return
+		return err
 	}
 	fmt.Println(ipRange.Start)
+	return nil
 }
 
-func ActionForCIDR(input string) {
+func ActionForCIDR(input string) error {
 	_, ipnet, err := net.ParseCIDR(input)
 	if err != nil {
```

Comment:
Although the output was correct for `ipinfo tool lower` but modified for the sake of consistency.